roles/install_java/tasks/main.yml

```diff
@@ -16,8 +16,21 @@
     path: "/usr/lib/jvm/{{ java_package }}"
   register: java_package_info
 
+- name: Check if JRE exists
+  ansible.builtin.stat:
+    path: "{{ java_package_info.stat.lnk_source }}/jre"
+  register: java_jre
```

Comment:
Since Java 11, the [JRE is no longer shipped with the JDK](https://www.oracle.com/java/technologies/javase/11-relnote-issues.html#:~:text=In%20Windows%20and%20macOS%2C%20installing,Only%20the%20JDK%20is%20offered.)